com.liferay.blade.cli/src/com/liferay/blade/cli/InitCommand.java

```diff
@@ -51,6 +51,8 @@ public InitCommand(blade blade, InitOptions options) throws Exception {
 	}
 
 	public void execute() throws IOException {
+		GradleTooling.setRefresh(_options.refresh());
```

Comment:
We don't need a "setRefresh".  just add a boolean in the findLatestAvailableArtifact() method.